client/src/components/About/about.css

```diff
@@ -0,0 +1,5 @@
+.About{
```

Comment:
small letters for class name (also use dashes, so `about-page` as opposed to `aboutPage`)